tests/shap/test_log.py

```diff
@@ -0,0 +1,93 @@
+import mlflow
+import shap
+import numpy as np
+import sklearn
+from mlflow.utils.environment import _mlflow_conda_env
+
+
+def get_sklearn_conda_env():
+    """
+    :return: The default Conda environment for MLflow Models produced by calls to
+             :func:`save_model()` and :func:`log_model()`.
+    """
+    pip_deps = ["shap=={}".format(shap.__version__)]
+
+    return _mlflow_conda_env(
+        additional_conda_deps=["scikit-learn={}".format(sklearn.__version__)],
+        additional_pip_deps=pip_deps,
+        additional_conda_channels=None,
+    )
```

Comment:
```suggestion
```

Do we use this function in `tests/shap/test_log.py`?

Reply:
Removed this function, it was not being used